src/app/search/query.js

```diff
@@ -6,4 +6,71 @@ export const getRequestParams = (query = {}) => {
   return { q, field, workId };
 };
 
-export default getRequestParams;
+/**
+ * Format a single field query or field queries based on string sent. Strings
+ * are broken up by word to AND them together by adding a separate field/value
+ * object for each word given.
+ *
+ * @param {string} queryString
+ * @param {string} field
+ *
+ * @returns {string}
+ */
+const addFieldQuery = (queryString, field = 'keyword') => {
+  if (!queryString) {
+    throw new Error('A valid query string must be passed');
+  }
+  let fieldQuery = [];
+  // Strip punctuation and process spaces as plus signs for final split.
+  const queryArr = queryString.replace(/[.,\/#!$%\^&;:{}=\-_`~()]/g, '').trim().replace(/\s+/g, '+').split('+');
```

Comment:
I agree with stripping these, but I can see some characters especially commas and colons being used in valid searches?

Also, an extreme edge case, but searching for _just_ punctuation returns nothing but it appears in the URL. Should probably throw the same error as an empty search string?

Reply:
As far as the first question, I've decided to alter the field query build a bit by sending a query like "hell +poetry" multiple terms instead of multiple field objects. I think that will work better.

I've added a TODO in comments for the latter. I need another PR, I think to manage errors and display them back to the user. Strange queries that become empty after processing should just return an empty result set for now.